services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsCredentialsProvider.java

```diff
@@ -83,6 +96,14 @@ public void close() {
         sessionCache.close();
     }
 
+    public Duration staleTime() {
+        return staleTime;
+    }
+
+    public Duration prefetchTime() {
```

Comment:
Can we add javadoc: The amount of time, relative to STS token expiration, that the cached credentials are considered close to stale and should be updated.

Reply:
Done